images/alpine/lib/scripts/spigot_init.sh

```diff
@@ -17,18 +24,40 @@ if [ ! -e $SPIGOT_HOME/eula.txt ]; then
   fi
 fi
 
-#only build if jar file does not exist
-if [ ! -f $SPIGOT_HOME/spigot.jar ]; then
+# Some variables are mandatory.
+if [ -z "$REV" ]; then
+    REV="latest"
+fi
+
+# Some variables depend on other variables.
+
+# Creeper block disable is a feature of the Essentials plugin.
+if [ -n "$CREEPERBLOCKDISABLE" ]; then
+    if [ "$CREEPERBLOCKDISABLE" = "true" ]; then
+	     ESSENTIALS=true
+    fi
+fi
+
+# Force rebuild of spigot.jar if REV is latest.
+rm -f $SPIGOT_HOME/spigot-latest.jar
```

Comment:
I think that I'd like to have a way to prevent a rebuild all the time when using latest. 
It should be an ENV variable set to `true` to decide whether or not we should for a rebuild

```suggestion
 if [ -n "$FORCELATESTREBUILD" ]; then
   if [ "$FORCELATESTREBUILD" = "true" ]; then
     rm -f $SPIGOT_HOME/spigot-latest.jar
   fi
 fi
```

Reply:
I'm concerned about changing the default behavior in this situation.  Users who wish to prevent a rebuild while using the latest revision can use the specific value for that revision instead of the floating target of 'latest'.  With this change, if neither `REV` nor `FORCELATESTREBUILD` are set in the config, the server will be built precisely once -- the first time the server starts.  This makes 'latest' much less useful as a revision.  Is there something I'm missing?